tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging.py

```diff
@@ -21,6 +21,7 @@
 import pytest
 
 from assertpy import assert_that
+from pcluster.cloudwatch_logs.utils import get_log_group_name
```

Comment:
I would allow to create dependencies between integration tests and the code defined in the cli. One of the reasons being we can move these tests in a separate package and the other being that sometimes we want to use the test from the develop branch and run them against an older version of the cli deployed through PyPI which might not contain the required dependencies.

(p.s. same needs to be done for the dcv test in case you want to open a quick patch :D)

Reply:
Done. And noted ;).